flow/templates/print_status_stack.jinja

```diff
@@ -0,0 +1,21 @@
+{% extends "print_status.jinja" %}
+{% block detailed %}
+{% set format_head = format_job_id  + '%s' + format_label %}
+{% set format_operation = ("%%1s %%-%ss %%3s  " | format(column_width_operation,)) %}
+{% for job in jobs %}
+{% if parameters %}
+{% set para_output = ns.format_parameters | format(*job['parameters'].values()) %}
+{% endif %}
+{{ format_job_id | format(job['job_id'],) }}{{para_output}}{{ '%s' | format(job['labels']|join(', ')) }}
+{% for key,value in job['operations'].items() %}
+{% if (alias[value['scheduler_status']] != 'U' or value['eligible']) or all_ops%}
+{{ format_operation | bold_font(value['eligible'],pretty) | format(value | get_operation_status(pretty),key,'['+alias[value['scheduler_status']]+']',) }}
+{% endif %}
+{% endfor %}
+{% endfor %}
+{% if pretty %}
+Legend: ○:ineligible ●:eligible ▹:active ▸:running ✔:completed
+{% else %}
+Legend: -:ineligible +:eligible *:active >:running X:completed
```

Comment:
I think it would be better to have a `{{ legend }}` variable that we pre-populate.